src/internal/common/utils/bool_file_store_test.go

```
@@ -0,0 +1,225 @@
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestReadBoolFile(t *testing.T) {
+	// Create a temporary directory for test files
+	tempDir, err := os.MkdirTemp("", "bool_file_test")
```

golangci-lint (1.64.8)

os.MkdirTemp() could be replaced by t.TempDir() in TestReadBoolFile

(usetesting)